articles/operator-insights/operator-insights-faq.yml

```diff
@@ -20,7 +20,7 @@ sections:
           Azure Operator Insights doesn't move or store customer data out of the region it's deployed in.
           
           The only exceptions to this are:
-          - Data processing for the Azure Operator Insights Copilot feature (currently in preview). This is because the Azure Open AI service used by the Copilot feature is not available in all regions yet. Data processing may happen cross region but within the same geo.
+          - Data processing for the Azure Operator Insights Copilot feature (currently in preview). This is because the Azure OpenAI service used by the Copilot feature is not available in all regions yet. Data processing may happen cross region but within the same geo.
```

Comment:
```suggestion
          - Data processing for the Azure Operator Insights Copilot feature (currently in preview). This is because Azure OpenAI Service used by the Copilot feature is not available in all regions yet. Data processing may happen cross region but within the same geo.
```